### PR TITLE
arm64: dts: qcom: msm8916-motorola-osprey: add sound

### PR DESCRIPTION
**Pull request by MartijnBraam**
Makes audio work, tested speaker, earpiece and headphone playback and I'm getting a signal from both built-in mics

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts [hunk 1]
```diff
@@ -212,6 +217,40 @@
...
+	status = "okay";
+
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&cdc_pdm_lines_act>;
+	pinctrl-1 = <&cdc_pdm_lines_sus>;
+
+	model = "msm8916";
+	audio-routing =
+		"AMIC1", "MIC BIAS External1",
+		"AMIC2", "MIC BIAS Internal2",
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts:230]: Hm, have you tried if headset microphones are working with this?

Comment (MartijnBraam) [reply arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts:230]: I sadly don't own a compatible headset with mic, but it seems connected like this in the schematic

File: arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts [hunk 2]
```diff
@@ -221,6 +260,14 @@
 	extcon = <&usb_id>;
 };
 
+&wcd_codec {
+	qcom,micbias-lvl = <2800>;
+	qcom,mbhc-vthreshold-low = <75 150 237 450 500>;
+	qcom,mbhc-vthreshold-high = <75 150 237 450 500>;
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts:266]: remove these 3

File: arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts (hunk 1 above)
Comment (Minecrell) [reply arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts:230]: remove this one since it doesn't seem to be used

**Comment by MartijnBraam** [comment]
Did the cleanup, audio still works

**Review by Minecrell — APPROVED** [review]

**Pull request merged**